packages/cloud_firestore/cloud_firestore/lib/src/collection_reference.dart

```diff
@@ -2,14 +2,14 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// @dart=2.9
-
 part of cloud_firestore;
 
 /// A [CollectionReference] object can be used for adding documents, getting
 /// [DocumentReference]s, and querying for documents (using the methods
 /// inherited from [Query]).
+@immutable
 class CollectionReference extends Query {
+  @override
```

Comment:
This override is not required, and I think it will be showing a linting 'info' about not overriding fields.